app/Services/BaseService.php

```diff
@@ -182,6 +182,10 @@ protected function buildParameters(): array
 
     protected function shortNameWithVersion(): string
     {
+        if (! preg_match('/v?(0|(?:[1-9]\d*))(?:\.(0|(?:[1-9]\d*))(?:\.(0|(?:[1-9]\d*)))?)/', $this->tag)) {
```

Comment:
Would you mind leaving a comment here about what this is for? Thanks for the catch; I wish I had even thought of this!

Reply:
Done 👍🏼